File: src/main/java/com/campool/model/Rental.java
package com.campool.model;

import java.time.LocalDate;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.springframework.format.annotation.DateTimeFormat;

@Getter
@Setter
@ToString
public class Rental {

    private int rentalId;

    private String title;

    private String userId;

    @DateTimeFormat(pattern = "yyyy-MM-dd")
    private LocalDate startDate;

    @DateTimeFormat(pattern = "yyyy-MM-dd")
    private LocalDate endDate;

    private int cost;

    private double longitude;

    private double latitude;

}

File: src/main/java/com/campool/model/Rental.kt
package com.campool.model

import java.time.LocalDate

data class Rental(
    val rentalId: Int,
    val title: String,
    val userId: String,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val cost: Int,
    val longitude: Double,
    val latitude: Double
)